db/migrate/20201015135045_backfill_recruitment_cycle_year_in_ucas_matches.rb

```diff
@@ -0,0 +1,10 @@
+class BackfillRecruitmentCycleYearInUCASMatches < ActiveRecord::Migration[6.0]
+  def change
+    start_of_2021_cycle = Date.new(2020, 10, 13)
+    ucas_matches_in_2020_cycle = UCASMatch.where('created_at < ?', start_of_2021_cycle)
+
+    ucas_matches_in_2020_cycle.update_all(recruitment_cycle_year: 2020)
```

Comment:
Are we sure we have no 2021 matches? (we could have sent some relevant applications, as we send everything right now)

Reply:
The last update was on 21 September so I think it's safe.

Reply:
I will amend to update all that were created after the new recruitment cycle date to `2021`